etl/steps/data/garden/who/2024-07-26/mortality_database.py

```diff
@@ -50,7 +50,8 @@ def tidy_sex_dimension(tb: Table) -> Table:
     Improve the labelling of the sex column
     """
     sex_dict = {"All": "Both sexes", "Female": "Females", "Male": "Males", "Unknown": "Unknown sex"}
-    tb["sex"] = tb["sex"].replace(sex_dict, regex=False)
+    # tb["sex"] = tb["sex"].replace(sex_dict, regex=False)
```

Comment:
Do you anticipate that we will need to revert these changes again at some point? Otherwise I would take these comments out (they are a bit confusing imo)